FNA.sln

```diff
@@ -9,8 +9,8 @@ Global
 		Release|x86 = Release|x86
 		Debug|x64 = Debug|x64
 		Release|x64 = Release|x64
-		Debug|AnyCPU = Debug|AnyCPU
-		Release|AnyCPU = Release|AnyCPU
+		Debug|AnyCPU = Debug|Any CPU
+		Release|AnyCPU = Release|Any CPU

```

Comment:
Does it still work after making this change?

```suggestion
		Debug|Any CPU = Debug|Any CPU
		Release|Any CPU = Release|Any CPU
```

Reply:
Let me check

Reply:
It does work too, indeed.